modules/programs/borgmatic.nix

```diff
@@ -154,7 +190,7 @@ let
   writeConfig = config:
     generators.toYAML { } (removeNullValues ({
       source_directories = config.location.sourceDirectories;
-      repositories = config.location.repositories;
+      repositories = cleanRepositories config.location.repositories;
```

Comment:
this should no be necessary if using `apply` as suggested

Reply:
I'll remove it